lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -59,22 +59,16 @@
     version_added: "2.2"
   wait:
     description:
-      - Wait for the copied AMI to be in state 'available' before returning.
+      - wait for the copied AMI to be in state 'available' before returning.
     required: false
-    default: "no"
-    choices: [ "yes", "no" ]
-  wait_timeout:
-    description:
-      - How long before wait gives up, in seconds.
-    required: false
-    default: 1200
```

Comment:
Is there documentation on the official way to deprecate this? This parameter is no longer required as boto3 handles this with [Waiters](https://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.Waiter.ImageAvailable).
```
Polls EC2.Client.describe_images() every 15 seconds until a successful state is reached. An error is returned after 40 failed checks.
```